src/backend/dx12/src/device.rs

```diff
@@ -3777,11 +3777,17 @@ impl d::Device<B> for Device {
     }
 
     fn start_capture(&self) {
```

Comment:
I think all of the `start_capture` and `end_capture` should themselves be `unsafe fn`, to match the other methods. But this is slightly above the scope of this PR. If you feel positive about doing this change, let's do it right here.